### DISCUSSION TITLE
Name explanation

### DISCUSSION THREAD

**Discussion by s0xDk** [utmapp/UTM#4223] [General]
What UTM stands for?
Would you mind adding the abbreviation description to the FAQ?

Thank you!

**Comment by osy** [comment]
The Alan Turing quote is a hint. Look it up on Wikipedia 🙂

**Comment by s0xDk** [comment]
😀 it was too easy after the hint.

https://en.m.wikipedia.org/wiki/Universal_Turing_machine

thanks!
